src/Model/PowerElectronics/SystemModelPowerElectronics.hpp

```diff
@@ -12,6 +12,49 @@
 
 namespace GridKit
 {
+  /**
+   * Writes a vector to a file in Matrix Market format
+   *
+   * @param vec The vector to write
+   * @param filename The name of the output file
+   * @param header Additional header information/comments
+   * @return true if the write was successful, false otherwise
+   */
+  template <typename T>
+  bool writeVectorToMatrixMarket(const std::vector<T>& vec, const std::string& filename, const std::string& header)
+  {
```

Comment:
Why is this function of `bool` type? Writing matrix was `void`. Also, it may be a little redundant to print a message that printing was successful.